recipes/ndg-httpsclient/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ndg-httpsclient" %}
+{% set pypi_name = "ndg_httpsclient" %}
+{% set version = "0.4.2" %}
+{% set hash_type = "sha256" %}
+{% set hash = "580987ef194334c50389e0d7de885fccf15605c13c6eecaabd8d6c43768eb8ac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ pypi_name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - ndg_httpclient = ndg.httpsclient.utils:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pyasn1
+    - pyopenssl
+
+test:
+  imports:
+    - ndg
+    - ndg.httpsclient
+    - ndg.httpsclient.test
+
+about:
+  home: https://github.com/cedadev/ndg_httpsclient
+  license: BSD 3-Clause
```

Comment:
Must have missed/forgotten the inclusion issues. Have added PR ( https://github.com/goanpeca/staged-recipes/pull/1 ) against your branch in the hopes that it helps. Seems to work locally.